components/item-info.js

```diff
@@ -144,6 +144,11 @@ export default function ItemInfo ({
           </Link>}
         {me && !item.meSats && !item.position &&
           !item.mine && !item.deletedAt && <DontLikeThisDropdownItem id={item.id} />}
+        {me && item?.noteId && (
+          <Dropdown.Item onClick={() => window.open(`https://nostr.band/${item.noteId}`, '_blank')}>
```

Comment:
Any reason for nostr.band specifically?

I've seen there are nostr wrapper clients (or however they are called) that let you pick which client you want to use. [nostr.com](https://nostr.com/) is one of them (actually don't know any else, lol)

So maybe use nostr.com if there is no reason against it? I don't know much about the nostr ecosystem. But that one lets you pick whatever client you want to use while nostr.band does not afaict:

_nostr.band_
![2023-12-07-175942_1920x1080_scrot](https://github.com/stackernews/stacker.news/assets/27162016/f536bfb5-c435-4701-9388-4de78dbcfa55)

_nostr.com_
![2023-12-07-175926_1920x1080_scrot](https://github.com/stackernews/stacker.news/assets/27162016/4b5ed206-0b02-4f6f-a894-326a202865d4)

Reply:
oh, and please use `noopener` as the third argument to the `window.open` call. [seems to be good security practice.](https://stackoverflow.com/questions/40593632/use-window-open-but-block-use-of-window-opener)